website/docs/terms/dataframe.md

```diff
@@ -0,0 +1,24 @@
+---
+id: dataframe
+title: Data Frame
```

Comment:
```suggestion
title: Data frame
```
Adjusting capitalization to be consistent with other glossary pages :)

Reply:
The consensus online seems actually to be with `DataFrame`. I checked Pandas, Snowpark and Spark and they all use it in one word, with the `F` uppercase